src/Orckestra.Composer.Website/UI.Package/Templates/KvaItems.hbs

```diff
@@ -1,58 +1,114 @@
 <div data-templateid="KvaItems">
     {{#each KeyVariantAttributeItems}}
         {{#if Values}}
-            <div class="mb-3  btn-multiline" data-qa="product-variants" data-propertyname="{{PropertyName}}" data-propertydatatype="{{PropertyDataType}}">
+          <div class="mb-3" data-qa="product-variants" data-propertyname="{{PropertyName}}" data-propertydatatype="{{PropertyDataType}}">
 
             <h5 class="font-weight-bold">{{DisplayName}}</h5>
+            <div class="kva-list variant-kva">
+              {{#if_eq PropertyName "Colour"}}
```

Comment:
at some point we need to migrate this to razor  :)

Reply:
I tried but failed :(